### PR TITLE
Pages design completed

### PR DESCRIPTION
**Pull request by animeshgarai09**

### CODE REVIEW

File: src/components/video-card/video-card.js [hunk 1]
```diff
@@ -6,12 +6,12 @@ import { BsThreeDotsVertical, FiClock, BiListPlus, RiShareForwardLine } from "@i
...
 import { useState } from "react"
 
-const VideoCard = () => {
+const VideoCard = ({ orientation }) => {
 
     const [drop, setDrop] = useState(false)
 
     return (
-        <div className={`${styles.container} ${styles.vertical}`}>
+        <div className={`${styles.container} ${orientation ? styles.horizontal : styles.vertical}`}>
```
Comment (nayyyhaa) [inline src/components/video-card/video-card.js:14]: good use of ternary operator to show conditional styling.

File: src/pages/watch-later/watch-later.js [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
     return (
-        <div>WatchLater</div>
+        <div className={styles.container}>
+            <SidePane title="Watch Later..." time="30 Minutes" count={30} />
+            <div className={styles.main}>
+                <VideoCard orientation="horizontal" />
+                <VideoCard orientation="horizontal" />
+                <VideoCard orientation="horizontal" />
+                <VideoCard orientation="horizontal" />
+                <VideoCard orientation="horizontal" />
```
Comment (nayyyhaa) [inline src/pages/watch-later/watch-later.js:13]: Since this is a basic layout-it's fine, later you can map through the repeating component.

**Review by nayyyhaa — APPROVED** [review]
not much to comment on since this only includes basic layout. Your UI looks awesome! Nice work 💯

**Review by hsnice16 — APPROVED** [review]
Nice work, bro. Couldn't find anything to suggest, and the UI is great as always.

**Pull request merged**